devices.js

```diff
@@ -17319,27 +17319,28 @@ const devices = [
     },
 
     {
-        zigbeeModel: ['HAN'],
-        model: 'Datek_HAN',
+        zigbeeModel: ['Meter Reader'],
```

Comment:
Shouldn't we also keep the original `zigbeeModel` here in case someone has a device with an old firmware?

Reply:
@Koenkk The product are released to the market now in April. Nobody will have fw < 0.4. But I will raise the issue with the developers of the manufacturer after easter and ask them kindly to add a searchable model name. I really agree about the naming, and this was no better solution than the previous `Datak HAN`

Please merge this so all proto's will work (they told be I should have the only proto-version running 0.2.